addons/crm/tests/test_crm_lead_merge.py

```diff
@@ -84,6 +85,71 @@ def test_initial_data(self):
         self.assertEqual(self.lead_w_email_lost.user_id, self.env['res.users'])
         self.assertEqual(self.lead_w_email_lost.team_id, self.sales_team_1)
 
+    @users('user_sales_manager')
+    @mute_logger('odoo.models.unlink')
+    def test_lead_merge_address_not_propagated(self):
+        """The empty field of the address should not be filled if other fields are not empty."""
+        initial_address = {
+            'street': 'Test street',
+            'street2': 'Test street2',
+            'city': 'Test City',
+            'zip': '5000',
+            'state_id': False,
+            'country_id': self.env.ref('base.be'),
+        }
+        self.lead_w_contact.write(initial_address)
+
+        # 5 non-empty address field like for the destination field, should be ignored
```

Comment:
They are ignored because we have "5 data (like initial address) but less prior", is it right ? Could gain a comment by the way :D